### DISCUSSION TITLE
DLNA (Digital Media Renderer) Media players not available

### DISCUSSION THREAD

**Discussion by timvdsm** [music-assistant/support#773] [Q&A] [answered]
### What version of Music Assistant has the issue?

2022.07

### The problem

The players from the DLNA Digital Media Renderer integration are not available. I have 2 media players from this integration, however they are not available in the Music Assistant config. Any idea of there is another solution to integrate Dlna players?

### How to reproduce

Setup Music Assistant

### Additional information

Integration:
https://www.home-assistant.io/integrations/dlna_dmr

### What version of Home Assistant Core are your running

2022.07

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

ODROID

**Comment by marcelveldt** [comment, marked as answer]
If a player is not available to select within the Music Assistant options, it means the player does not support one of the mandatory services, especially "play_media". If you can actually play back a radio stream on the device something is wrong in how the device is being detected.

**Reply by OzGav** [reply]
Please see the trouble shooting FAQ for help with this

**Reply by timvdsm** [reply]
Indeed, my bad. It seems the DLNA integration with my Samsung TV’s is not working properly. I’m not able to play music through the DLNA integration itself, so it will definitely not work in MA. However in LMS the DLNA/UPNP bridge works perfectly with my Samsung TV’s. But that’s of course related to the DLNA integration itself.
Is anyone able to use their Samsung TV’s (tizen) in MA? I’ve tried the HACS Samsung Smart TV integration, but also no luck.
